### PR TITLE
feat(Tooltip): Add accessibility attributes and keyboard handlers

### PR DESCRIPTION
**Pull request by sophieH29**
Add ARIA attributes and keyboard handlers to `Tooltip` component

TODO: 
- update CHANGELOG

### CODE REVIEW

**Comment by jurokapsiar** [comment]
looks good, we'll do some tests with screen readers today

File: docs/src/examples/components/Tooltip/Types/TooltipExample.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Button, Tooltip } from '@stardust-ui/react'
 
 const TooltipExample = () => (
   <Tooltip content="Hello from tooltip!">
-    <Button icon="expand" />
+    <Button content="Click me!" />
```
Comment (layershifter) [inline docs/src/examples/components/Tooltip/Types/TooltipExample.tsx]:
```suggestion
    <Button>Click me!</Button>
```

It's an example for children API...

File: packages/react/src/components/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -116,16 +119,33 @@ export default class Tooltip extends AutoControlledComponent<TooltipProps, Toolt
...
+      e.preventDefault()
+    },
+  }
+
+  static getAutoControlledStateFromProps(
+    props: TooltipProps,
+    state: TooltipState,
+  ): Partial<TooltipState> {
+    return {
+      tooltipId: getOrGenerateIdFromShorthand('tooltip-content-', props.content, state.tooltipId),
```
Comment (layershifter) [inline packages/react/src/components/Tooltip/Tooltip.tsx]:
```suggestion
      tooltipId: getOrGenerateIdFromShorthand('tooltip-content-', props.content, state.contentId),
```

It looks like a `contentId`...

File: packages/react/src/lib/getOrGenerateIdFromShorthand.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { ShorthandValue } from '../types'
+
+const getOrGenerateIdFromShorthand = (
+  idPrefix: string,
```
Comment (layershifter) [inline packages/react/src/lib/getOrGenerateIdFromShorthand.ts]:
```suggestion
  prefix: string,
```

nit: feel free to take own decision 😺

**Review by layershifter — APPROVED** [review]

**Review by jurokapsiar — APPROVED** [review]

**Pull request merged**